buildSrc/src/main/groovy/nullaway.jacoco-conventions.gradle

```diff
@@ -0,0 +1,46 @@
+// Mostly taken from official Gradle sample: https://docs.gradle.org/current/samples/sample_jvm_multi_project_with_code_coverage.html
```

Comment:
@lazaroclapp I presume we shouldn't slap an Uber license header on this file, since it's almost all taken from official sample code?

Reply:
Um, not sure what the official response of Uber OSS is going to be here, but, for now, I'd assume: a) if it's from the docs, then we have permission to use this code? b) it's probably fine to have it with that comment?

cc: @tyvsmith ?

Reply:
Yes, this is fine.

Reply:
Thanks @tyvsmith!  Will add the header then